ipv-stub/src/endpoints/ipv-reverification.ts

```diff
@@ -43,6 +44,18 @@ async function get(
     return invalidAccessTokenResult();
   }
 
+  logger.info(
+    "reverification result here: \n" + JSON.stringify(reverification)
+  );
+
+  if (!reverification.success) {
+    logger.info(
+      "failed json result here: \n" +
+        JSON.stringify(failedJsonResult(500, reverification))
+    );
+    return failedJsonResult(500, reverification);
```

Comment:
I think this should return a 200 code, not 500.

The request and response was successful, but the response indicates an problem in the users IPV journey (not an internal server error etc)